cime_config/cesm/machines/config_batch.xml

```diff
@@ -111,7 +111,7 @@
     <batch_query args="-f" >qstat</batch_query>
     <batch_submit>qsub </batch_submit>
     <batch_directive>#PBS</batch_directive>
-    <jobid_pattern>^(\d+)\.</jobid_pattern>
+    <jobid_pattern>^(\S+)$</jobid_pattern>
```

Comment:
The jobid string returned from pbs comes in the form 1234.pbsserver   we were only using the 1234 part and removing the pbsserver part, it turns out that we need this part for Cheyenne and I don't think that it has a negative consequence to use the whole string everywhere.